src/Microsoft.DotNet.Darc/DarcLib/GitHubClient.cs

```diff
@@ -986,12 +986,12 @@ private async Task<IList<Check>> GetChecksFromChecksApiAsync(string owner, strin
                             switch (run.Conclusion?.Value)
                             {
                                 case CheckConclusion.Success:
+                                case CheckConclusion.Neutral:
```

Comment:
I think what we want is skipped. I don't really know what changing neutral to be successful would do

https://docs.github.com/en/graphql/reference/enums#checkrunstate

Reply:
I think neutral is what we would want in this case. The check is marked as completed, and in neutral state, if it was skipped:

https://[api.github.com/repos/dotnet/installer/check-suites/17545809960/check-runs](https://api.github.com/repos/dotnet/installer/check-suites/17545809960/check-runs)

```
"total_count": 13,
    "check_runs": [
        {
            "id": 17975599117,
            "name": "installer-source-build",
            "node_id": "CR_kwDOBjr6NM8AAAAEL23gDQ",
            "head_sha": "9f3f39b211f6c25a43d6f36756b650d344f1172f",
            "external_id": "",
            "url": "https://api.github.com/repos/dotnet/installer/check-runs/17975599117",
            "html_url": "https://github.com/dotnet/installer/runs/17975599117",
            "details_url": "https://dev.azure.com/dnceng-public/cbb18261-c48f-4abb-8651-8cdcb5474649/_build/definition?definitionId=233",
            "status": "completed",
            "conclusion": "neutral",
            "started_at": "2023-10-23T19:25:05Z",
            "completed_at": "2023-10-23T19:25:05Z",
            "output": {
                "title": "This check was skipped",
                "summary": "No pipeline [branch filters](https://go.microsoft.com/fwlink/?linkid=2084226) matched the modified branch.",
                "text": null,
                "annotations_count": 0,
                "annotations_url": "https://api.github.com/repos/dotnet/installer/check-runs/17975599117/annotations"
            },
```

Reply:
Ahh, so "this check was skipped" is just the title of the non-skipped check...